src/debug/ee/debugger.cpp

```diff
@@ -15306,6 +15306,13 @@ HRESULT Debugger::FuncEvalSetup(DebuggerIPCE_FuncEvalInfo *pEvalInfo,
         return CORDBG_E_ILLEGAL_AT_GC_UNSAFE_POINT;
     }
 
+    if (::GetSP(filterContext) != ALIGN_DOWN(::GetSP(filterContext), STACK_ALIGN_SIZE))
+    {
+        // SP is not aligned, we cannot do a FuncEval here
+        LOG((LF_CORDB, LL_INFO1000, "D::FEA: SP is unaligned"));
+        return CORDBG_E_FUNC_EVAL_BAD_START_POINT;
```

Comment:
@gregg-miskelly I chose this error code as it seemed reasonable.  Would you prefer a different error code?

Reply:
Sounds reasonable to me.